BlueprintUI/Sources/Layout/Column.swift

```diff
@@ -41,15 +41,39 @@ public struct Column: StackElement {
         }
     }
 
-    public var children: [(element: Element, traits: StackLayout.Traits, key: AnyHashable?)] = []
+    public var children: [StackLayout.Child] = []
 
-    public private(set) var layout = StackLayout(
+    public private(set) var layout: StackLayout = StackLayout(
         axis: .vertical,
-        alignment: ColumnAlignment.leading.stackAlignment
+        alignment: Column.ColumnAlignment.leading.stackAlignment
     )
 
     public init() {}
 
+    /// Initializer using result builder to declaritively build up a stack.
+    /// - Parameters:
+    ///   - alignment: Specifies how children will be aligned horizontally. Default: .leading
+    ///   - underflow: Determines the layout when there is extra free space available. Default: .spaceEvenly
+    ///   - overflow: Determines the layout when there is not enough space to fit all children as measured. Default: .condenseProportionally
+    ///   - minimumSpacing: Spacing in between elements. Default: 0
+    ///   - elements: A block containing all elements to be included in the stack.
+    /// - Note: If element is a plain Element, then that Element will be implicitly converted into a `StackLayout.Child` with default values
```

Comment:
Since we are introducing radically new API here, I think it would be helpful to have more in the docs, particularly explaining the way to add child traits.

```suggestion
    /// Creates a Column, using result builder syntax. Columns display a list of items in a vertical
    /// stack.
    ///
    ///     Column {
    ///         Label(text: "Welcome")
    ///
    ///         TextField(text: username)
    ///         TextField(text: password)
    ///
    ///         Button(
    ///             onTap: handleSignIn,
    ///             wrapping: Label(text: "Sign In")
    ///         )
    ///     }
    ///
    /// By default, each item in the column will be stretched or compressed with equal priority in
    /// the event of overflow or underflow. You can control this behavior by adding a
    /// `stackLayoutChild` modifier to an item.
    ///
    ///     Column {
    ///         ImportantHeader()
    ///             .stackLayoutChild(priority: .fixed)
    ///
    ///         LessImportantContent()
    ///     }
    ///
    /// You can also use this modifier to add keys and alignment guides. See `StackElement.add` for
    /// more information.
    ///
    /// ## See Also
    /// [StackElement.add()](x-source-tag://StackElement.add)
    ///
    /// - Parameters:
    ///   - alignment: Specifies how children will be aligned horizontally. Default: `.leading`
    ///   - underflow: Determines the layout when there is extra free space available. Default:
    ///     `.spaceEvenly`
    ///   - overflow: Determines the layout when there is not enough space to fit all children as
    ///     measured. Default: `.condenseProportionally`
    ///   - minimumSpacing: Spacing in between elements. Default: `0`
    ///   - elements: A block containing all elements to be included in the stack.
```